Parsely/wp-parsely
# Rename the main class file to class-parsely.php & source it in the plugin entry file

## Description

Akin to #261, this is an effort to fix the style issues surfaced in Automattic/vip-go-mu-plugins#2084

I wanted to split this one out from that change so we can discuss it independently if desired.

We've discussed doing a lot more to split the main class out, but I think keeping this change small and focused on the naming convention & addressing #96 with focused changes is probably for the best.

### The Change

Since this diff is a bit hard to grok in the web UI, I'll describe what I did:

* `git mv wp-parsely.php class-parsely.php`
* `touch wp-parsely.php`
* Cut the plugin header from the class file
* Paste the plugin header to the new main file
* Add a `require` statement at the bottom of the new main file
* Stage the changes, commit, & push

## To Test

This should really change nothing at all in the way the plugin works. It should register, activate, operate as before. Automated tests should function as before.
